### ISSUE TITLE
Unable to start after upgrading from 0.11 to 6.0

### ISSUE THREAD

**Issue by mingfang** [confluentinc/ksql#6304]
This is the error on startup

```
[2020-09-26 03:11:59,105] ERROR Error during restore (io.confluent.ksql.rest.server.computation.CommandRunner)                                                                                             
 org.apache.kafka.common.errors.SerializationException: Error deserializing key/value for partition _confluent-ksql-ksqldb-server-confluentinc-example_command_topic-0 at offset 0. If needed, please seek  
 past the record to continue consumption.                                                                                                                                                                   
 Caused by: org.apache.kafka.common.errors.SerializationException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "orReplace" (class io.confluent.ksql.execution.ddl. 
 commands.CreateStreamCommand), not marked as ignorable (6 known properties: "schema", "sourceName", "topicName", "timestampColumn", "windowInfo", "formats"])                                              
  at [Source: (byte[])"{"statement":"CREATE STREAM MOVIE_TICKET_SALES (title VARCHAR, sale_ts VARCHAR, ticket_total_value INT)\n    WITH (KAFKA_TOPIC='movie-ticket-sales',\n          PARTITIONS=1,\n      
      VALUE_FORMAT='avro');","streamsProperties":{"auto.offset.reset":"latest"},"originalProperties":{"ksql.extension.dir":"ext","ksql.streams.cache.max.bytes.buffering":"10000000","ksql.transient.prefix 
 ":"transient_","ksql.persistence.wrap.single.values":"true","ksql.authorization.cache.expiry.time.secs":"30","ksql.sch"[truncated 2516 bytes]; line: 1, column: 3003] (through reference chain: io.conflue 
 nt.ksql.rest.server.computation.Command["plan"]->io.confluent.ksql.engine.KsqlPlanV1["ddlCommand"]->io.confluent.ksql.execution.ddl.commands.CreateStreamCommand["orReplace"])                             
 Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "orReplace" (class io.confluent.ksql.execution.ddl.commands.CreateStreamCommand), not marked as ignorable  
 (6 known properties: "schema", "sourceName", "topicName", "timestampColumn", "windowInfo", "formats"])                                                                                                     
  at [Source: (byte[])"{"statement":"CREATE STREAM MOVIE_TICKET_SALES (title VARCHAR, sale_ts VARCHAR, ticket_total_value INT)\n    WITH (KAFKA_TOPIC='movie-ticket-sales',\n          PARTITIONS=1,\n      
      VALUE_FORMAT='avro');","streamsProperties":{"auto.offset.reset":"latest"},"originalProperties":{"ksql.extension.dir":"ext","ksql.streams.cache.max.bytes.buffering":"10000000","ksql.transient.prefix 
 ":"transient_","ksql.persistence.wrap.single.values":"true","ksql.authorization.cache.expiry.time.secs":"30","ksql.sch"[truncated 2516 bytes]; line: 1, column: 3003] (through reference chain: io.conflue 
 nt.ksql.rest.server.computation.Command["plan"]->io.confluent.ksql.engine.KsqlPlanV1["ddlCommand"]->io.confluent.ksql.execution.ddl.commands.CreateStreamCommand["orReplace"])                             
     at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)                                                           
```

**Comment by apurvam** [comment]
cc @stevenpyzhang this would be expected in ‘rollbacks’ before 0.12 right? (This is effectively a rollback from 0.11 to 0.10).

**Comment by apurvam** [comment]
Specifically, we would expect 0.12 to start in degraded state even if the command topic had commands issued with 0.13.

**Comment by mingfang** [comment]
sorry I was confused by the version numbering system and thought it was an upgrade.
Is there a version lookup table, from ksqldb version -> cp-ksqldb version?

**Comment by stevenpyzhang** [comment]
Yeah, the 0.10.0 has no rollback considerations built in (such as handling deserialization errors) so this deserialization is uncaught and crashes the server on start up.

@mingfang the version interoperability matrix between CP and standalone ksqldb is here
https://docs.confluent.io/current/installation/versions-interoperability.html#ksqldb

**Comment by mingfang** [comment]
Thanks everyone for the information.
This is not an issue so I'm closing.

**Comment by mingfang** [comment]
Why is Confluent Platform 6 using such an old version of ksqldb?
So CP 6 just released and is using ksqldb 0.10, which is already old.
But then ksqldb 0.12 is coming out soon.
Are we supposed to use the CP version or the ksqldb version?
